modules/gapi/include/opencv2/gapi/oak/oak.hpp

```diff
@@ -89,28 +89,55 @@ G_API_OP(GSobelXY, <GFrame(GFrame, const cv::Mat&, const cv::Mat&)>, "org.opencv
     }
 };
 
+G_API_OP(GCopy, <GFrame(GFrame)>, "org.opencv.oak.copy") {
+    static GFrameDesc outMeta(const GFrameDesc& in) {
+        return in;
+    }
+};
```

Comment:
Is a new operation really required?
https://docs.opencv.org/4.5.5/d6/d91/group__gapi__transform.html#ga1dc3b1987d6685689ecc0d590abddba8